easybuild/easyconfigs/m/MaSuRCA/MaSuRCA-3.2.2-foss-2016a.eb

```diff
@@ -0,0 +1,44 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2017 University of Geneva
+# Authors::   Yann Sagon <yann.sagon@unige.ch>
+# License::   MIT/GPL
+# $Id$
+#
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'MaSuRCA'
+version = '3.2.2'
+
+homepage = 'http://www.genome.umd.edu/masurca.html'
+
+description = '''MaSuRCA is whole genome assembly software. It combines the efficiency of the de Bruijn graph  
+ and Overlap-Layout-Consensus (OLC) approaches. MaSuRCA can assemble data sets containing 
+ only short reads from Illumina sequencing or a mixture of short reads and long reads 
+ (Sanger, 454, Pacbio and Nanopore).'''
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+buildopts="install-special"
+start_dir="global-1"
+
+#need a temporary url to download it. Do it manually.
+source_urls = ['http://www.genome.umd.edu/masurca_form.html']
```

Comment:
don't specify this via `source_urls`, but just as a comment, there's no point in letting EasyBuild attempt a download